test/bdd/pkg/vc/vc_steps.go

```diff
@@ -242,41 +263,85 @@ func (e *Steps) verifyCredential(validVCArgKey string) error {
 	return nil
 }
 
-func (e *Steps) checkProfileResponse(expectedProfileResponseArgkey string,
+func (e *Steps) checkProfileResponse(expectedProfileResponseName string,
 	profileResponse *operation.ProfileResponse) error {
-	expectedProfileResponse := operation.ProfileResponse{}
+	if profileResponse.Name != expectedProfileResponseName {
+		return fmt.Errorf("expected %s but got %s instead", expectedProfileResponseName, profileResponse.Name)
+	}
+
+	expectedProfileResponseDID := "did:peer:22"
```

Comment:
it should be constant, same applies bellow

Reply:
done

Reply:
done. I also split up the `checkVC` function.